scheduler/views.py

```diff
@@ -177,5 +177,8 @@ def log_in(request: HttpRequest) -> HttpResponse:
 def edit(request: HttpRequest) -> HttpResponse:
     """Render the edit page"""
     if request.method == 'POST':
-        return HttpResponseRedirect(reverse('index'))
+        form = EditForm(request.POST)
+        if form.is_valid():
+            return HttpResponseRedirect(reverse('index'))
```

Comment:
Czy te eventy nie powinny się jakoś jednak zapisywać do bazy? Bo na ten moment dodanie/zmiana eventu nie mają efektywnie żadnego skutku... :/

Reply:
Zapisywanie do bazy jest w osobnym tasku. No i przydałoby się wtedy chociaż ID pobrać do formularza, czyt. uzupełnić go, co jest jeszcze innym taskiem

Reply:
A po stworzeniu nowego eventu? Tu trzeba wygenerować nowe chyba

Reply:
Planowałem zrobić to w tym ZIEL-46, bo w opisie było tylko o dodaniu formularza. Plus wtedy ten 46 polegałby na dodaniu w tej samej metodzie ifa czy został podany ID czy nie

Reply:
Ma to sens :D